libp2p/Common.h

```diff
@@ -114,6 +114,8 @@ using CapDesc = std::pair<std::string, unsigned>;
 using CapDescSet = std::set<CapDesc>;
 using CapDescs = std::vector<CapDesc>;
 
+std::ostream& operator<<(std::ostream& _out, CapDesc const& _capDesc);
```

Comment:
Does logging not work without this?
We have a  boost.log output operator for `std::pair` (outputs it like "(first, second)"), but I'm not sure whether it's picked up for the alias of pair.